go/kbfs/data/path_part_string.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package data
+
+import (
+	"encoding"
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+const (
+	prefixToSkipObfsucation = ".kbfs_"
```

Comment:
is there any chance we'll want to expand this to more than one thing such that this should be an array of strings?

Reply:
I think it's very low.  in the 4+ years of KBFS, we've only had this one special prefix.  if we ever do, we can always implement the array at that time instead of wasting CPU on it now.